MetaMask/metamask-extension
# feat: :sparkles: remove the alerts tab

## **Description**

This PR removes the Alerts tab from the settings page.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27033?quickstart=1)

## **Related issues**

[Fixes:](https://github.com/MetaMask/MetaMask-planning/issues/3212)

## **Manual testing steps**

1. Go to the settings page
2. Verify that the alerts tab is not present

## **Screenshots/Recordings**

N/A

### **Before**

<img width="374" alt="Screenshot 2024-09-10 at 16 21 21" src="https://github.com/user-attachments/assets/a13be357-c6d6-4b28-a92a-2aaea4f044e2">

### **After**



<img width="376" alt="Screenshot 2024-09-10 at 16 20 03" src="https://github.com/user-attachments/assets/28ce9ead-8f4b-46df-9d77-61ff74380954">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
